cfpb/cfgov-refresh
# CCDB landing: Migrate icon to SVG

## Changes

- Convert cf-icon into SVG.

## Testing

1. Run server and visit http://localhost:8000/data-research/consumer-complaints/
2. Inspect the page and find `cf-notification` class in the markup and turn off the `display: none` CSS rule. See that gold icon is present.

## Screenshots

<img width="533" alt="Screen Shot 2019-05-10 at 6 05 18 PM" src="https://user-images.githubusercontent.com/704760/57559090-337d3e00-734e-11e9-911d-6b3841aae26a.png">
